config/application.rb

```
@@ -30,6 +30,26 @@ class Application < Rails::Application
 
     # Raise errors when an unpermitted param is sent to a controller action
     ActiveRecord::Base.send(:include, ActiveModel::ForbiddenAttributesProtection)
+
+    config.app_uri = URI.parse ENV.fetch('APP_URI')
+
+    # ActionMailer Config with Mandrill from Mailchimp for transactional mail
+    config.action_mailer.delivery_method = ENV.fetch('MAIL_DELIVERY_METHOD', :smtp).to_sym
+    config.action_mailer.smtp_settings = {
+      address:   'smtp.mandrillapp.com',
+      port:      587,
+      enable_starttls_auto: true,
+      user_name: ENV.fetch('MANDRILL_USERNAME'),
+      password:  ENV.fetch('MANDRILL_APIKEY'),
+      authentication: :plain,
+      domain: config.app_uri.host
+    }
+
+    config.action_mailer.perform_deliveries    = true
+    config.action_mailer.raise_delivery_errors = false
```

That would make sense. I think I left it this way because we normally use Queue Classic, which eats all the errors anyway.